### PR TITLE
feat(ingest/looker): Record observed lineage timestamps for Looker and LookML sources

### PR DESCRIPTION
**Pull request by ANich**
Adds timestamps to the Looker and LookML sources for lineage between Explores and Views as well as between Views and SQL tables/views. 

## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
~- [ ] Links to related issues (if applicable)~
- [x] Tests for the changes have been added/updated (if applicable)
~- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.~
~- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)~

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py [hunk 1]
```diff
@@ -841,6 +843,10 @@ def _to_metadata_events(  # noqa: C901
                     UpstreamClass(
                         dataset=view_urn,
                         type=DatasetLineageTypeClass.VIEW,
+                        auditStamp=AuditStamp(
+                            time=int(observed_lineage_ts.timestamp() * 1000),
+                            actor="urn:li:corpuser:datahub",
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py:848]: it'd be good to extract this out into a constant

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py [hunk 2]
```diff
@@ -21,6 +21,7 @@
 )
 
 import pydantic
+from datahub.metadata.com.linkedin.pegasus2avro.common import AuditStamp
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py:24]: looks like linter doesn't like the import sorting here

**Review by hsheth2 — CHANGES_REQUESTED** [review]

**Comment by laulpogan** [comment]
@ANich - we haven't seen movement on this PR for a little while- are you still interested in contributing? If so, please let us know within the week, or we'll need to close this one for inactivity.

**Comment by ANich** [comment]
> @ANich - we haven't seen movement on this PR for a little while- are you still interested in contributing? If so, please let us know within the week, or we'll need to close this one for inactivity.

Hey @laulpogan @anshbansal I'll pick this back up in the next couple days if that's okay. Will ping for a re-review

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py [hunk 3]
```diff
@@ -786,6 +789,7 @@ def _to_metadata_events(  # noqa: C901
         if self.upstream_views is not None:
             assert self.project_name is not None
             upstreams = []
+            observed_lineage_ts = datetime.datetime.now()
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py:792]:
```suggestion
            observed_lineage_ts = datetime.datetime.now(tz=timezone.utc)
```

File: metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py [hunk 4]
```diff
@@ -1615,11 +1617,16 @@ def _get_upstream_lineage(
 
         # Generate the upstream + fine grained lineage objects.
         upstreams = []
+        observed_lineage_ts = datetime.now()
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py:1620]: needs tz

**Review by hsheth2 — COMMENTED** [review]
We should to add tz to now() calls, but otherwise lgtm

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py (hunk 3 above)
Comment (ANich) [reply metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py]: Updated! Thanks for the review

**Comment by anshbansal** [comment]
cypress test failures unrelated so merging.

**Pull request merged**